Source/Lib/Decoder/Codec/EbDecParseObu.c

```diff
@@ -2330,3 +2333,39 @@ EbErrorType decode_multiple_obu(EbDecHandle *dec_handle_ptr, const uint8_t *data
     }
     return status;
 }
+
+EB_API EbErrorType eb_get_sequence_info(
+    const uint8_t *obu_data,
+    uint32_t       size,
+    SeqHeader     *sequence_info)
+{
+    const uint8_t* frame_buf = obu_data;
```

Comment:
Since this is an api function, we should check the input parameters.

Reply:
done